src/Dialog/dialog.scss

```diff
@@ -0,0 +1,52 @@
+/*
+ * Copyright (c) 2021 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-4-Clause
+ */
+
+@import '../variables';
+
+.dialog {
+    .modal-content {
+        border: 0 transparent;
+        box-shadow: 0px 3px 6px #00000029;
+        background: #ffffff 0% 0% no-repeat padding-box;
+    }
+
+    .modal-header {
+        height: 40px;
+        border: 0 transparent;
+        background: $gray-50;
```

Comment:
Could just let this be a `.css` file. To use a variable you'd just use the syntax:

```css
background: var(--gray-50);
```

Reply:
I'd prefer to not mix css and scss files in the projects, though I don't see a major difference in this case in general.
I'll keep it as scss for now.